charts/agent/templates/configmap-deployment.yaml

```
@@ -43,6 +43,18 @@ data:
 {{- if .Values.global.sysdig.tags }}
     tags: {{ include "agent.tags" . }}
 {{- end }}
+{{- if and (or (.Values.proxy.httpProxy | default .Values.global.proxy.httpProxy) (.Values.proxy.httpsProxy | default .Values.global.proxy.httpsProxy)) (not .Values.sysdig.settings.http_proxy) }}
+    proxy:
```

```suggestion
    http_proxy:
```

Based on our [docs](https://docs.sysdig.com/en/docs/installation/configuration/sysdig-agent/tune-agent/enable-http-proxy-for-agents/#examples), this key is `http_proxy` and not `proxy`.